AlpsSDK/Repositories/Devices/MobileDeviceRepository.swift

```diff
@@ -11,11 +11,27 @@ import Foundation
 import Foundation
 import Alps
 
+let kMainDeviceFile = "kMainDeviceFile.Alps"
+let kMobileDevicesFile = "kMobileDevicesFile.Alps"
+
 final class MobileDeviceRepository: AsyncCreateable, AsyncReadable, AsyncDeleteable {
     typealias DataType = MobileDevice
     
-    private(set) var items = [MobileDevice]()
-    private(set) var main: Device?
+    private(set) var items = [MobileDevice]() {
+        didSet {
+            _ = PersistenceManager.save(object: self.items.map { $0.encodableMobileDevice }, to: kMobileDevicesFile)
```

Comment:
We could remove the `_ =` ? 
First time, I see didSet keyword. It is interesting 👍

Reply:
Very convinient.

Important part - to make it really work you need value type inside. For class it won't trigger when you edit properties of the class. For struct and swift collections it works like a charm.